Tasks/AzureResourceGroupDeploymentV2/Tests/EnablePrereq.ts

```diff
@@ -42,9 +42,9 @@ let a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
 
 tr.setAnswers(a);
 
-tr.registerMock('vsts-task-lib/toolrunner', require('vsts-task-lib/mock-toolrunner'));
-tr.registerMock('azure-arm-rest/webRequestUtility', require('./mock_node_modules/webRequestUtility'));
-tr.registerMock('azure-arm-rest/azure-arm-compute', require('./mock_node_modules/azure-arm-compute'));
-tr.registerMock('azure-arm-rest/azure-arm-network', require('./mock_node_modules/azure-arm-network'));
-tr.registerMock('azure-arm-rest/azure-arm-resource', require('./mock_node_modules/azure-arm-resource'));
+tr.registerMock('azure-pipelines-task-lib/toolrunner', require('azure-pipelines-task-lib/mock-toolrunner'));
+tr.registerMock('azure-arm-rest-v2/webRequestUtility', require('./mock_node_modules/webRequestUtility'));
```

Comment:
Why this change? azure-arm-rest-v2 might have breaking changes compated to azure-arm-rest

Reply:
azure-arm-rest-v2 contains upgraded version of azure-pipelines-task-lib. 
azure-arm-rest depends on vsts-tasks-lib & azure-arm-rest-v2 depends on azure-pipelines-task-lib.

So if you want to upgrade the task you should also use the upgraded common libraries which are using upgraded version of azure-pipelines-task-lib. o/w it may cause compatibility issues.

Reply:
The goal here is to update task lib version. If you use v2 of common utilities they might have side impact of some common functionality being broken. And then we need to test for those.
Do you know what is delta between v1 and v2?

Reply:
there is not much difference b/w azure-arm-rest and azure-arm-rest-v2. azure-arm-rest-v2 is using upgraded azure-pipelines-task-lib